platform-hub-api/spec/controllers/kubernetes/tokens_controller_spec.rb

```diff
@@ -199,7 +201,7 @@
       {
         kind: kind,
         cluster_name: @cluster.name,
-        groups: 'group1,group2,group3',
```

Comment:
`groups` come as a string from the UI when creating a token so would probably make sense to leave it like that.

Reply:
It can be provided as either a string or array – (e.g. for robot tokens it's an Array). And note that with upcoming changes it'll also be an array for user tokens too. We test out the string/array handling in the model tests, and also in `PUT update` test later. So I made this one an array just to make it read slightly better.